forge-gui/res/cardsfolder/upcoming/the_capitoline_triad.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Capitoline Triad
+ManaCost:10
+Types:Legendary Creature God Artificer
+PT:7/7
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Those Who Came Before — This spell costs {1} less to cast for each historic card in your graveyard. (Artifacts, legendaries, and Sagas are historic.)
+A:AB$ Effect | Cost$ ExileFromGrave<1+/Card.Historic+withTotalCMCEQ30/historic cards from your graveyard with total mana value 30 or greater> | Name$ Emblem - The Capitoline Triad | Image$ emblem_the_capitoline_triad | StaticAbilities$ STSetPT | Stackable$ False | Duration$ Permanent | AILogic$ Always | SpellDescription$ You get an emblem with "Creatures you control have base power and toughness 9/9."

```

Comment:
this still needs engine support though

Reply:
And that should probably wait until we're SURE it's real :)

Reply:
@Simisays can we skip this file for a later Commit?